test/e2e-tests.sh

```diff
@@ -106,7 +106,7 @@ function run_e2e_tests() {
   kubectl label namespace $2 istio-injection=enabled --overwrite
   local options=""
   (( EMIT_METRICS )) && options="-emitmetrics"
-  report_go_test -v -tags=e2e -count=1 ./test/$1 -dockerrepo gcr.io/knative-tests/test-images/$1 ${options}
+  report_go_test -v -tags=e2e -count=1 -timeout=20m ./test/$1 -dockerrepo gcr.io/knative-tests/test-images/$1 ${options}
```

Comment:
I would recommend using some environment variable for test timeout and default the value of variable to 20m.  Probably document the variable and usage in test docs.

Reply:
yes, makes sense. will make the change.

Reply:
Not an issue anymore. This commit from yesterday increased the timeout to 20 mins - https://github.com/knative/serving/pull/1702/commits/fbfa23dbf40e7231f9d076635b68a059d6ca3f69